src/components/Modal/Modal.css

```diff
@@ -0,0 +1,53 @@
+.Modal__root {
+  height: 100vh;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+.Modal__form {
+  background-color: var(--color-dark-secondary);
+  max-width: 24rem;
+  height: 100%;
+  margin: 0 auto;
+  padding: 2rem 1rem;
+  border-radius: 4px;
+}
+
+.Modal__form > input {
+  padding: 10px 12px;
+  width: 100%;
+  margin-bottom: 1rem;
+  border-radius: 4px;
+  font-size: 1rem;
+  outline: none;
+  border: none;
+  background-color: var(--color-dark);
+  color: #f7f7f7;
```

Comment:
You can use variables here also, as u already used the above code. 
If the same color is not repeated thn its okay